parquet/src/test/java/org/apache/iceberg/parquet/TestBloomRowGroupFilter.java

```diff
@@ -951,6 +968,72 @@ public void testStructFieldNotEq() {
     }
   }
 
+  @Test
+  public void testStructFieldWithSpecialCharacterLt() {
```

Comment:
@huaxingao, can you review these tests?

Reply:
@rdblue These tests look good. Thanks for pinging me.